### PR TITLE
Proposed Docs for new integration AlphaESS

### PR DESCRIPTION
**Pull request by CharlesGillanders**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/62801
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3024
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/alphaess.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Alpha Ess
+description: Instructions on how to integrate Alpha ESS Solar Storage with Home Assistant
+  - Sensor
+ha_category:
+  - Energy
+ha_iot_class: Cloud Polling
+ha_release: 2022.1
```
Comment (frenck) [inline source/_integrations/alphaess.markdown]: This version won't exist; the first upcoming release will be 2022.2

File: source/_integrations/alphaess.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ha_codeowners:
+  - '@CharlesGillanders'
+---
+
+This integration pulls data from an [Alpha ESS](https://www.alpha-ess.com/) Energy Storage Solution.
+  
+## Integration Entities
+  
+Each added configuration entry will create the following sensors:  
+(All measurements are in Watt hours (Wh) unless otherwise noted)
```
Comment (frenck) [inline source/_integrations/alphaess.markdown]: Let's move this below the list.

**Review by frenck — CHANGES_REQUESTED** [review]
This page misses instructions on configuring/setting up this integration.

File: source/_integrations/alphaess.markdown [hunk 3]
```diff
@@ -15,11 +15,15 @@ ha_codeowners:
 ---
 
 This integration pulls data from an [Alpha ESS](https://www.alpha-ess.com/) Energy Storage Solution.
+
+## Configuration
+
+The integration can be entirely setup from within the GUI.  Add the integration to Home Assistant and provide the username and password needed to logon to the Alpha ESS website.  Any Alpha ESS energy storage units registered with Alpha ESS will be automatically discovered and added to Home Assistant.
```
Comment (frenck) [inline source/_integrations/alphaess.markdown]: Please see how other integration pages do this, or check our developer documentation (which also provides information on documenting).

Use: `{% include includes/config_flow.md %}`

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/alphaess.markdown (hunk 3 above)
Comment (CharlesGillanders) [reply source/_integrations/alphaess.markdown]: Thanks - I have updated the documentation to follow this standard.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**